public/API/API_GetUserRecentlyPlayedGames.php

```diff
@@ -22,15 +22,21 @@
  *    int        ScoreAchievedHardcore    number of points earned by the user in hardcore
  */
 
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Validator;
+
+$input = Validator::validate(Arr::wrap(request()->query()), [
+    'u' => 'required',
```

Comment:
Note that this could include a lookup:
```
'u' => 'required|exists:mysql_legacy.UserAccounts,User',
```
I opted to not do that to reduce db hits as user data is fetched after validation anyways if needed.

Reply:
I think it's reasonable to not validate the presence of the data at this point.

I've already noticed discrepancies between the various APIs when a non-existant user is queried. Most return an empty object, some return a 404 error, and one returns a fully populated (but zeroed out) response.